src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Tabs.css';
+
+export class Tabs extends React.Component {
+  state = {
+    focusedTitle: 'Tab 1',
+  }
```

Comment:
save this to App's state
mb it's better to save the whole object of selected tab, and not just title